src/esp/gfx/GenericDrawable.cpp

```diff
@@ -147,10 +150,14 @@ void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
     shader_->bindNormalTexture(*(materialData_->normalTexture));
   }
 
-  shader_->draw(mesh_);
+  shader_->draw(*mesh_);
```

Comment:
We should assert non-null before de-referencing here.

Maybe add a `getMesh()` in Drawable that does the assert (and make mesh_ private?). It makes me nervous that we're de-referencing this pointer in lots of places scattered throughout lots of files. Casual users won't understand that this pointer may be null for the obscure case of `createRenderer==false`.